src/engine/types/index.ts

```diff
@@ -60,11 +60,6 @@ export class Type<DefinitionType extends TypeDefinition> {
 export class DisplayType<DefinitionType extends DisplayTypeDefinition> extends Type<DefinitionType> {
     private priority: number = 0
 
-    constructor(engine: Engine, definition: DefinitionType) {
-        super(engine, definition)
-        this.priority = definition.PRIORITY ?? 0
-    }
-
```

Comment:
I hope it's not required to leave sth like xD
```
    constructor(engine: Engine, definition: DefinitionType) {
        super(engine, definition)
    }
```

Reply:
You removed assigning priority to this.priority, so it won't return the correct priority with GETPRIORITY?

Reply:
Hmm, that's right. Should `GETPRIORITY` be working before `init()`?

Reply:
It's not gonna be called before init() so no

Reply:
Could alternatively implement an `init()` here to be called at the end of `init()` in children.

```ts
init() {
    this.getRenderObject()!.zIndex = this.priority
    this.engine.sortObjects()
}
```

Reply:
What problem would that solve? 🤔

Reply:
Code duplication (by introducing more code duplication).  
OK, I guess it's not relevant.